src/search-schema/data/definitions/enums/assay_types.yaml

```diff
@@ -40,7 +40,7 @@ celldive_deepcell:
     alt-names: []
     primary: false
     contains-pii: false
-    vitessce-hints: ['codex', 'is_image', 'is_tiled']
+    vitessce-hints: ['sprm', 'anndata', 'multiscale', 'is_image', 'is_tiled']
```

Comment:
ok i forgot about `is_tiled` so we can drop `multiscale`

Reply:
Done.